crates/typst-library/src/layout/align.rs

```diff
@@ -102,41 +102,38 @@ impl Show for Packed<AlignElem> {
     }
 }
 
-/// Where to [align] something along an axis.
+/// 軸に沿って何かを[配置]($align)する位置。
 ///
-/// Possible values are:
-/// - `start`: Aligns at the [start]($direction.start) of the [text
-///   direction]($text.dir).
-/// - `end`: Aligns at the [end]($direction.end) of the [text
-///   direction]($text.dir).
-/// - `left`: Align at the left.
-/// - `center`: Aligns in the middle, horizontally.
-/// - `right`: Aligns at the right.
-/// - `top`: Aligns at the top.
-/// - `horizon`: Aligns in the middle, vertically.
-/// - `bottom`: Align at the bottom.
+/// 取りうる値は
```

Comment:
箇条書きの前に文として終わらせると、日本語として自然になると思います。

```suggestion
/// 取りうる値は以下の通りです。
```